pkg/controllers/disruption/consolidation_test.go

```diff
@@ -687,58 +745,78 @@ var _ = Describe("Consolidation", func() {
 							BlockOwnerDeletion: ptr.Bool(true),
 						},
 					}}})
-
-			nodePool2 := test.NodePool()
-			cloudProvider.InstanceTypesForNodePool[nodePool2.Name] = nil
-			ExpectApplied(ctx, env.Client, rs, pod, node, nodeClaim, nodePool, nodePool2)
+			ExpectApplied(ctx, env.Client, rs, pod, spotNode, spotNodeClaim, nodePool)
 
 			// bind pods to node
-			ExpectManualBinding(ctx, env.Client, pod, node)
+			ExpectManualBinding(ctx, env.Client, pod, spotNode)
 
 			// inform cluster state about nodes and nodeclaims
-			ExpectMakeNodesAndNodeClaimsInitializedAndStateUpdated(ctx, env.Client, nodeStateController, nodeClaimStateController, []*v1.Node{node}, []*v1beta1.NodeClaim{nodeClaim})
+			ExpectMakeNodesAndNodeClaimsInitializedAndStateUpdated(ctx, env.Client, nodeStateController, nodeClaimStateController, []*v1.Node{spotNode}, []*v1beta1.NodeClaim{spotNodeClaim})
 
 			fakeClock.Step(10 * time.Minute)
 
 			// consolidation won't delete the old nodeclaim until the new nodeclaim is ready
 			var wg sync.WaitGroup
 			ExpectTriggerVerifyAction(&wg)
-			ExpectMakeNewNodeClaimsReady(ctx, env.Client, &wg, cluster, cloudProvider, 1)
 			ExpectReconcileSucceeded(ctx, disruptionController, client.ObjectKey{})
 			wg.Wait()
 
-			// Process the item so that the nodes can be deleted.
-			ExpectReconcileSucceeded(ctx, queue, types.NamespacedName{})
-
-			// Cascade any deletion of the nodeclaim to the node
-			ExpectNodeClaimsCascadeDeletion(ctx, env.Client, nodeClaim)
-
-			// should create a new nodeclaim as there is a cheaper one that can hold the pod
-			nodeclaims := ExpectNodeClaims(ctx, env.Client)
-			nodes := ExpectNodes(ctx, env.Client)
-			Expect(nodeclaims).To(HaveLen(1))
-			Expect(nodes).To(HaveLen(1))
-
-			// Expect that the new nodeclaim does not request the most expensive instance type
-			Expect(nodeclaims[0].Name).ToNot(Equal(nodeClaim.Name))
-			Expect(scheduling.NewNodeSelectorRequirements(nodeclaims[0].Spec.Requirements...).Has(v1.LabelInstanceTypeStable)).To(BeTrue())
-			Expect(scheduling.NewNodeSelectorRequirements(nodeclaims[0].Spec.Requirements...).Get(v1.LabelInstanceTypeStable).Has(mostExpensiveInstance.Name)).To(BeFalse())
+			// shouldn't delete the node
+			Expect(ExpectNodes(ctx, env.Client)).To(HaveLen(1))
 
-			// and delete the old one
-			ExpectNotFound(ctx, env.Client, nodeClaim, node)
+			// Expect Unconsolidatable events to be fired
+			evts := recorder.Events()
+			_, ok := lo.Find(evts, func(e events.Event) bool {
```

Comment:
nit: consider in-lining 
```suggestion
			_, ok := lo.Find(recorder.Events(), func(e events.Event) bool {
```

Reply:
Addressed.